### PR TITLE
Bug 1296927, added specific sysconfig files

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1296927

Related to https://github.com/openshift/openshift-docs/pull/1864 and https://bugzilla.redhat.com/show_bug.cgi?id=1304855

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @jkaurredhat  @tnguyen-rh @bfallonf PTAL. Thanks!

File: install_config/http_proxies.adoc [hunk 1]
```diff
@@ -23,12 +23,18 @@ files.
 
 . Add the `*NO_PROXY*`, `*HTTP_PROXY*`, and `*HTTPS_PROXY*` environment variables
 ifdef::openshift-origin[]
-to each host's *_/etc/sysconfig/origin-master_* or
-*_/etc/sysconfig/origin-node_* file depending on the type of host:
+to each host's *_/etc/sysconfig/origin-master_* file (for single master
+configuration), or *_/etc/sysconfig/origin-master-api_* and
+*_/etc/sysconfig/origin-master-controllers_* files (for multi-master configuration),
+or *_/etc/sysconfig/origin-node_* file (for node configuration) depending on
+the type of host:
```
Comment (tnguyen-rh) [inline install_config/http_proxies.adoc]: The structure is basically:

    ...to each host's FILENAME (CONTEXT),
    or FILENAME (CONTEXT),
    or FILENAME and FILENAME (CONTEXT)
    depending on...

I would remove the first "or" and add a comma after the last "(CONTEXT)":

    ...to each host's FILENAME (CONTEXT),
    FILENAME (CONTEXT),
    or FILENAME and FILENAME (CONTEXT),
    depending on...

File: install_config/http_proxies.adoc [hunk 2]
```diff
@@ -23,12 +23,18 @@ files.
...
 endif::[]
 ifdef::openshift-enterprise[]
-to each host's *_/etc/sysconfig/atomic-openshift-master_* or
-*_/etc/sysconfig/atomic-openshift-node_* file depending on the type of host:
+to each host's *_/etc/sysconfig/atomic-openshift-master_* file (for single
+master configuration), or *_/etc/sysconfig/atomic-openshift-master-api_* and
+*_/etc/sysconfig/atomic-openshift-master-controllers_* files (for multi-master
+configuration), or *_/etc/sysconfig/atomic-openshift-node_* file (for node
+configuration) depending on the type of host:
```
Comment (tnguyen-rh) [inline install_config/http_proxies.adoc]: Likewise.

**Comment by tnguyen-rh** [comment]
@ahardin-rh 
Just a small structural nit (x2), otherwise LGTM.

**Comment by ahardin-rh** [comment]
@jkaurredhat please let me know if this satisfies your request. Thank you!

**Comment by ahardin-rh** [comment]
Approved by Jaspreet via e-mail.

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../install_config/http_proxies.html[Working with HTTP Proxies]
|Added specific *_/etc/sysconfig_* files to the link:../admin_guide/managing_projects.html#configuring-hosts-for-proxies[Configuring Hosts for Proxies] section.
%

**Pull request merged**